portfolio/src/main/webapp/index.html

```diff
@@ -299,6 +304,110 @@ <h1>Free Time</h1>
     </div>
     -->
 
+    <section id="nationalParks">
+    <div class="wrapper2" id="nationalParks">
+        <h1>National Parks</h1>
+        <hr>
+        <br>
+        <h3 style="text-align: center">Here is a map of all the national parks I have visited:</h3>
```

Comment:
Move all of these style=".." rules to the .css file so all the styling is in one place.